engine-transactions/src/eip_4844.rs

```diff
@@ -0,0 +1,160 @@
+use crate::eip_2930::AccessTuple;
+use crate::Error;
+use alloc::vec::Vec;
+use aurora_engine_precompiles::secp256k1::ecrecover;
+use aurora_engine_types::types::{Address, Wei};
+use aurora_engine_types::{H256, U256};
+use rlp::{Decodable, DecoderError, Encodable, Rlp, RlpStream};
+
+/// Type indicator (per EIP-2718) for access list transactions
+/// [EIP-4844 Specification](https://eips.ethereum.org/EIPS/eip-4844#specification)
+pub const BLOB_TX_TYPE: u8 = 0x03;
```

Comment:
I would rename it to `TYPE_BYTE` for consistency.

Reply:
Actually, it align with the specification: https://eips.ethereum.org/EIPS/eip-4844#specification
I'm not sure that renaming it for "_just good look_" is good reason enough.

Reply:
Specification it's excellent, but **actually** we have  our own naming convention (such historically happens) for these constants: `eip_1559::TYPE_BYTE`, `eip_2930::TYPE_BYTE`
And I'm not sure that changing this convention to "_just to be like in the specification_" is a good reason enough.

Reply:
Changed: 5ba5c12abaa8356357d4f23aa216d88ef72fb39d